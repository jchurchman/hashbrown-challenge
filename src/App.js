import styled from 'styled-components'
import AppHeader from './components/AppHeader'
import SearchBar from './components/SearchBar'

const AppContainer = styled.div`
  display: flex;
  justify-content: center;
  position: absolute;
  top: 13em;
  left: 0;
  right: 0;
  padding: 2em;
  background-color: light-gray;
`

function App() {
  return (
    <AppContainer>
      <AppHeader />
      <SearchBar />
    </ AppContainer>
  );
}

export default App;
