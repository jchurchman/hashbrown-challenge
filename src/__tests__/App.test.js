import ShallowRenderer from 'react-test-renderer/shallow';
import App from '../App'

describe('App ', () => {
  it('renders as expected ', () => {
    const renderer = new ShallowRenderer()
    const tree = renderer
      .render(<App />)
    const output = renderer
      .getRenderOutput()

    expect(output).toMatchSnapshot()
  })
})