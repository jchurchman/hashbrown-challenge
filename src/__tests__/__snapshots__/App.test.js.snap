// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  renders as expected  1`] = `
<styled.div>
  <AppHeader />
  <SearchBar
    handleFormSubmit={[Function]}
  />
  <styled.div>
    
  </styled.div>
  
</styled.div>
`;
